common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1466,6 +1466,8 @@
 
     public static final KnownKey invite_ignore_x_alt_description = KnownKey.newKey(true);
 
+    public static final KnownKey password_contains_username = KnownKey.newKey(false);
```

Comment:
config name doesn't sound right about its conditional usage, should we change it to `password_contains_username_allowed` or `allow_username_within_password`